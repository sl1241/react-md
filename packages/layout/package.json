{
  "name": "@react-md/layout",
  "version": "5.1.4",
  "description": "This package is used to configure most of react-md as well as create some pre-defined layouts",
  "main": "./lib/index.js",
  "module": "./es/index.js",
  "types": "./types/index.d.ts",
  "sideEffects": [
    "dist/**/*"
  ],
  "author": "Mikkel Laursen <mlaursen03@gmail.com>",
  "repository": {
    "type": "git",
    "url": "https://github.com/mlaursen/react-md.git",
    "directory": "packages/layout"
  },
  "bugs": {
    "url": "https://github.com/mlaursen/react-md/issues"
  },
  "homepage": "https://react-md.dev/packages/layout/demos",
  "keywords": [
    "accessibility",
    "a11y",
    "react-md",
    "material design",
    "react",
    "layout",
    "configuration",
    "context providers",
    "setup",
    "component",
    "components"
  ],
  "license": "MIT",
  "dependencies": {
    "@react-md/app-bar": "workspace:../app-bar",
    "@react-md/dialog": "workspace:../dialog",
    "@react-md/divider": "workspace:../divider",
    "@react-md/form": "workspace:../form",
    "@react-md/icon": "workspace:../icon",
    "@react-md/link": "workspace:../link",
    "@react-md/list": "workspace:../list",
    "@react-md/menu": "workspace:../menu",
    "@react-md/sheet": "workspace:../sheet",
    "@react-md/states": "workspace:../states",
    "@react-md/theme": "workspace:../theme",
    "@react-md/tooltip": "workspace:../tooltip",
    "@react-md/transition": "workspace:../transition",
    "@react-md/tree": "workspace:../tree",
    "@react-md/typography": "workspace:../typography",
    "@react-md/utils": "workspace:../utils",
    "classnames": "^2.3.2"
  },
  "devDependencies": {
    "@types/react": "^18.0.29",
    "csstype": "^3.1.1",
    "react": "^18.2.0",
    "react-dom": "^18.2.0",
    "react-router-dom": "^6.9.0"
  },
  "peerDependencies": {
    "react": ">= 16.14",
    "react-dom": ">= 16.14"
  },
  "peerDependenciesMeta": {
    "react": {
      "optional": false
    },
    "react-dom": {
      "optional": false
    }
  },
  "publishConfig": {
    "access": "public"
  }
}
