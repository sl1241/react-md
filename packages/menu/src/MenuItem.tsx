

import { ListItem } from "@react-md/list"
import { useKeyboardFocusableElement } from "@react-md/utils"

import type { MenuItemProps } from "./types"

/**
 * This component is used as an "action" within a `Menu`/`DropdownMenu` that
 * implements some keyboard focus behavior. This component should generally have
 * an `onClick` event handler.
 *
 * @remarks \@since 5.0.0
 */
export const MenuItem = ({ className, children, role = "menuitem", tabIndex = -1, ref: nodeRef, ...props }: MenuItemProps) => {
  const ref = useKeyboardFocusableElement(nodeRef as any)
  return (
    <ListItem
      {...props}
      ref={ref}
      //@ts-ignore
      role={role}
      tabIndex={tabIndex}
      className={["rmd-menu-item", className]}
    >
      {children}
    </ListItem>
  )
}

