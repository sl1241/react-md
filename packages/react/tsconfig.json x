{
  "extends": "../../tsconfig.base.json",
  "compilerOptions": {
    "outDir": "./dist",
    "rootDir": "./src",
    "declarationDir": "./dist/types",
    "composite": true,
    "skipLibCheck": true,
    "skipDefaultLibCheck": true,
    "jsx": "preserve",
    "jsxImportSource": "voby",
    "types": [
      "voby/dist/jsx/jsx"
    ],
  },
  "exclude": [
    "vite.config.ts",
"vite.config.umd.ts",
    "types",
  ],
}
