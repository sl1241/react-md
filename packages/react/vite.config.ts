import { defineConfig } from 'vite'

const config = defineConfig({
    build: {
        minify: false,
        lib: {
            entry: ["./src/index.ts"],
            name: "react",
            formats: ['cjs', 'es', 'umd'],
            fileName: (format: string, entryName: string) => `${entryName}.${format}.js`
        },
        // rollupOptions: {
        //   output: {
        //     preserveModules: true,
        //   }
        // },
    },


    esbuild: {
        jsx: 'automatic',
    },

    css: {
        modules: {
            scopeBehaviour: 'global',
            localsConvention: 'camelCase',
        }
    },
})



export default config
