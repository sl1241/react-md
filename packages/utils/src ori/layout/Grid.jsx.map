{"version":3,"file":"Grid.jsx","sourceRoot":"","sources":["Grid.tsx"],"names":[],"mappings":";;;;;;;;;;;;;;;;;;;;;;AAEA,OAAO,EAAE,QAAQ,EAAE,YAAY,EAAE,UAAU,EAAE,cAAc,EAAE,MAAM,OAAO,CAAC;AAC3E,CAAC;AAED,OAAO,EAAE,GAAG,EAAE,MAAM,QAAQ,CAAA;AAC5B,OAAO,EAAE,UAAU,EAAE,MAAM,sBAAsB,CAAA;AACjD,OAAO,EAAE,QAAQ,EAAE,MAAM,YAAY,CAAA;AAErC;;;;;;;GAOG;AACH,MAAM,CAAC,IAAM,gBAAgB,GAAG,iBAAiB,CAAA;AAEjD;;;;;GAKG;AACH,MAAM,CAAC,IAAM,eAAe,GAAG,mBAAmB,CAAA;AAkGlD,IAAM,KAAK,GAAG,GAAG,CAAC,UAAU,CAAC,CAAA;AAE7B;;;;;;GAMG;AACH,MAAM,CAAC,IAAM,IAAI,GAAG,UAAU,CAA4B,SAAS,IAAI,CACnE,EAgBC,EACD,GAAG;;IAhBC,IAAA,KAAK,WAAA,EACL,SAAS,eAAA,EACT,QAAQ,cAAA,EACR,aAAa,EAAb,KAAK,mBAAG,KAAK,KAAA,EACb,mBAAmB,EAAnB,WAAW,mBAAG,KAAK,KAAA,EACnB,gBAAgB,EAAhB,QAAQ,mBAAG,KAAK,KAAA,EAChB,OAAO,aAAA,EACP,YAAY,kBAAA,EACZ,aAAa,mBAAA,EACb,cAAc,oBAAA,EACd,mBAAmB,yBAAA,EACnB,OAAO,aAAA,EACP,MAAM,YAAA,EACN,YAAY,kBAAA,EACT,KAAK,cAfZ,gMAgBC,CADW;IAIN,IAAA,KAAmD,UAAU,EAAE,EAA7D,OAAO,aAAA,EAAE,QAAQ,cAAA,EAAE,SAAS,eAAA,EAAE,cAAc,oBAAiB,CAAA;IAErE,IAAM,WAAW,GAAG,oBAChB,OAAO,EAAE,CAAC,OAAO,KAAK,CAAC,IAAI,OAAO,CAAC,IAAI,SAAS,EAChD,mBAAmB,EAAE,YAAY;YAC7B,CAAC,CAAC,mCAA4B,YAAY,YAAS;YACnD,CAAC,CAAC,SAAS,IACZ,KAAK,gBACP,gBAAgB,IACb,CAAC,OAAO,IAAI,YAAY,CAAC;QACzB,CAAC,QAAQ,IAAI,aAAa,CAAC;QAC3B,CAAC,cAAc,IAAI,mBAAmB,CAAC;QACvC,CAAC,SAAS,IAAI,cAAc,CAAC;QAC7B,OAAO,KACV,eAAe,IAAG,MAAM,MACX,CAAA;IAClB,IAAM,eAAe,GAAG,EAAE,CAAC,KAAK,CAAC,EAAE,YAAY,EAAE,OAAO,KAAK,CAAC,EAAE,CAAC,EAAE,SAAS,CAAC,CAAA;IAE7E,IAAI,WAAW,IAAI,cAAc,CAAC,QAAQ,CAAC,EAAE;QACzC,IAAM,KAAK,GAAG,QAAQ,CAAC,IAAI,CAAC,QAAQ,CAAC,CAAA;QAErC,OAAO,YAAY,CAAC,KAAK,EAAE;YACvB,KAAK,wBAAO,WAAW,GAAK,KAAK,CAAC,KAAK,CAAC,KAAK,CAAE;YAC/C,SAAS,EAAE,EAAE,CAAC,eAAe,EAAE,KAAK,CAAC,KAAK,CAAC,SAAS,CAAC;SACtB,CAAC,CAAA;KACvC;IAED,IAAI,OAAO,GAAG,QAAQ,CAAA;IACtB,IAAI,KAAK,IAAI,QAAQ,EAAE;QACnB,OAAO,GAAG,QAAQ,CAAC,GAAG,CAClB,QAAQ,EACR,UAAC,KAAK,IAAK,OAAA,KAAK,IAAI,CAAC,QAAQ,CAAC,KAAK,CAAC,CAAC,KAAK,CAAC,CAAC,CAAC,KAAK,CAAC,EAAE,QAAQ,CAAC,EAAnD,CAAmD,CACjE,CAAA;KACJ;IAED,OAAO,CACH,CAAC,GAAG,CAAC,IAAI,KAAK,CAAC,CAAC,GAAG,CAAC,CAAC,GAAG,CAAC,CAAC,KAAK,CAAC,CAAC,WAAW,CAAC,CAAC,SAAS,CAAC,CAAC,eAAe,CAAC,CACrE;YAAA,CAAC,OAAO,CACZ;QAAA,EAAE,GAAG,CAAC,CACT,CAAA;AACL,CAAC,CAAC,CAAA"}