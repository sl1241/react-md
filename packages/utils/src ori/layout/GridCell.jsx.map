{"version":3,"file":"GridCell.jsx","sourceRoot":"","sources":["GridCell.tsx"],"names":[],"mappings":";;;;;;;;;;;;;;;;;;;;;;AACA,OAAO,EAAE,QAAQ,EAAE,YAAY,EAAE,UAAU,EAAE,cAAc,EAAE,MAAM,OAAO,CAAC;AAC3E,CAAC;AAED,OAAO,EAAE,GAAG,EAAE,MAAM,QAAQ,CAAA;AAC5B,OAAO,EAAE,UAAU,EAAE,MAAM,sBAAsB,CAAA;AAyFjD,IAAM,KAAK,GAAG,GAAG,CAAC,UAAU,CAAC,CAAA;AAE7B,MAAM,CAAC,IAAM,QAAQ,GAAG,UAAU,CAC9B,SAAS,QAAQ,CACb,EAgBC,EACD,GAAG;;IAhBC,IAAA,KAAK,WAAA,EACL,SAAS,eAAA,EACT,KAAK,WAAA,EACL,QAAQ,cAAA,EACC,WAAW,aAAA,EACV,YAAY,cAAA,EACd,UAAU,YAAA,EACT,WAAW,aAAA,EACV,YAAY,cAAA,EACd,UAAU,YAAA,EAClB,KAAK,WAAA,EACL,MAAM,YAAA,EACN,OAAO,aAAA,EACP,YAAY,kBAAA,EACT,KAAK,cAfZ,2JAgBC,CADW;IAIN,IAAA,KAAmD,UAAU,EAAE,EAA7D,OAAO,aAAA,EAAE,QAAQ,cAAA,EAAE,SAAS,eAAA,EAAE,cAAc,oBAAiB,CAAA;IAErE,IAAI,OAAO,GAAG,WAAW,CAAA;IACzB,IAAI,QAAQ,GAAG,YAAY,CAAA;IAC3B,IAAI,MAAM,GAAG,UAAU,CAAA;IACvB,IAAI,OAAO,GAAG,WAAW,CAAA;IACzB,IAAI,QAAQ,GAAG,YAAY,CAAA;IAC3B,IAAI,MAAM,GAAG,UAAU,CAAA;IACvB,IAAM,KAAK,GACP,CAAC,OAAO,IAAI,KAAK,CAAC;QAClB,CAAC,QAAQ,IAAI,MAAM,CAAC;QACpB,CAAC,cAAc,IAAI,YAAY,CAAC;QAChC,CAAC,SAAS,IAAI,OAAO,CAAC,CAAA;IAE1B,IAAI,KAAK,EAAE;QACP,CACI,KAMA,KAAK,QANgB,EAArB,OAAO,mBAAG,WAAW,KAAA,EACrB,KAKA,KAAK,SALkB,EAAvB,QAAQ,mBAAG,YAAY,KAAA,EACvB,KAIA,KAAK,OAJc,EAAnB,MAAM,mBAAG,UAAU,KAAA,EACnB,KAGA,KAAK,QAHgB,EAArB,OAAO,mBAAG,WAAW,KAAA,EACrB,KAEA,KAAK,SAFkB,EAAvB,QAAQ,mBAAG,YAAY,KAAA,EACvB,KACA,KAAK,OADc,EAAnB,MAAM,mBAAG,UAAU,KAAA,CACb,CAAA;KACb;IAED,IAAM,SAAS,cACX,eAAe,EAAE,QAAQ,EACzB,aAAa,EAAE,MAAM,EACrB,YAAY,EAAE,QAAQ,EACtB,UAAU,EAAE,OAAO,CAAC,CAAC,CAAC,eAAQ,OAAO,CAAE,CAAC,CAAC,CAAC,MAAM,IAC7C,KAAK,CACX,CAAA;IACD,IAAM,aAAa,GAAG,EAAE,CACpB,KAAK,CAAC,MAAM;QACR,GAAC,UAAG,OAAO,CAAE,IAAG,OAAO;YACzB,EACF,SAAS,CACZ,CAAA;IAED,IAAI,KAAK,IAAI,cAAc,CAAC,QAAQ,CAAC,EAAE;QACnC,IAAM,KAAK,GAAG,QAAQ,CAAC,IAAI,CAAC,QAAQ,CAAC,CAAA;QACrC,OAAO,YAAY,CAAC,KAAK,EAAE;YACvB,KAAK,wBAAO,KAAK,CAAC,KAAK,CAAC,KAAK,GAAK,SAAS,CAAE;YAC7C,SAAS,EAAE,EAAE,CAAC,aAAa,EAAE,KAAK,CAAC,KAAK,CAAC,SAAS,CAAC;SACpB,CAAC,CAAA;KACvC;IAED,OAAO,CACH,CAAC,GAAG,CAAC,IAAI,KAAK,CAAC,CAAC,GAAG,CAAC,CAAC,GAAG,CAAC,CAAC,KAAK,CAAC,CAAC,SAAS,CAAC,CAAC,SAAS,CAAC,CAAC,aAAa,CAAC,CACjE;gBAAA,CAAC,QAAQ,CACb;YAAA,EAAE,GAAG,CAAC,CACT,CAAA;AACL,CAAC,CACJ,CAAA"}