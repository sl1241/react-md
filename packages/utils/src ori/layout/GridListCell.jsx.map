{"version":3,"file":"GridListCell.jsx","sourceRoot":"","sources":["GridListCell.tsx"],"names":[],"mappings":";;;;;;;;;;;AACA,OAAO,EAAE,QAAQ,EAAE,YAAY,EAAE,UAAU,EAAE,cAAc,EAAE,MAAM,OAAO,CAAC;AAC3E,CAAC;AAED,OAAO,EAAE,GAAG,EAAE,MAAM,QAAQ,CAAA;AAiB5B,IAAM,KAAK,GAAG,GAAG,CAAC,eAAe,CAAC,CAAA;AAElC,MAAM,CAAC,IAAM,YAAY,GAAG,UAAU,CAClC,SAAS,YAAY,CACjB,EAAgE,EAChE,GAAG;IADD,IAAA,SAAS,eAAA,EAAE,QAAQ,cAAA,EAAE,cAAc,EAAd,MAAM,mBAAG,KAAK,KAAA,EAAE,aAAa,EAAb,KAAK,mBAAG,KAAK,KAAA,EAAK,KAAK,cAA9D,4CAAgE,CAAF;IAG9D,IAAM,aAAa,GAAG,EAAE,CAAC,KAAK,CAAC,MAAM,EAAE,EAAE,MAAM,QAAA,EAAE,CAAC,EAAE,SAAS,CAAC,CAAA;IAC9D,IAAI,KAAK,IAAI,cAAc,CAAC,QAAQ,CAAC,EAAE;QACnC,IAAM,KAAK,GAAG,QAAQ,CAAC,IAAI,CAAC,QAAQ,CAAC,CAAA;QACrC,OAAO,YAAY,CAAC,KAAK,EAAE;YACvB,SAAS,EAAE,EAAE,CAAC,aAAa,EAAE,KAAK,CAAC,KAAK,CAAC,SAAS,CAAC;SACpB,CAAC,CAAA;KACvC;IAED,OAAO,CACH,CAAC,GAAG,CAAC,IAAI,KAAK,CAAC,CAAC,GAAG,CAAC,CAAC,GAAG,CAAC,CAAC,SAAS,CAAC,CAAC,aAAa,CAAC,CAC/C;gBAAA,CAAC,QAAQ,CACb;YAAA,EAAE,GAAG,CAAC,CACT,CAAA;AACL,CAAC,CACJ,CAAA"}