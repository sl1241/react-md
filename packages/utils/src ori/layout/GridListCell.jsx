var __rest =
  (this && this.__rest) ||
  function (s, e) {
    var t = {};
    for (var p in s)
      if (Object.prototype.hasOwnProperty.call(s, p) && e.indexOf(p) < 0)
        t[p] = s[p];
    if (s != null && typeof Object.getOwnPropertySymbols === 'function')
      for (var i = 0, p = Object.getOwnPropertySymbols(s); i < p.length; i++) {
        if (
          e.indexOf(p[i]) < 0 &&
          Object.prototype.propertyIsEnumerable.call(s, p[i])
        )
          t[p[i]] = s[p[i]];
      }
    return t;
  };
import { Children, cloneElement, forwardRef, isValidElement } from 'react';
import { bem } from '../bem';
var block = bem('rmd-grid-list');
export var GridListCell = forwardRef(function GridListCell(_a, ref) {
  var className = _a.className,
    children = _a.children,
    _b = _a.square,
    square = _b === void 0 ? false : _b,
    _c = _a.clone,
    clone = _c === void 0 ? false : _c,
    props = __rest(_a, ['className', 'children', 'square', 'clone']);
  var cellClassName = cn(block('cell', { square: square }), className);
  if (clone && isValidElement(children)) {
    var child = Children.only(children);
    return cloneElement(child, {
      className: cn(cellClassName, child.props.className),
    });
  }
  return (
    <div {...props} ref={ref} className={cellClassName}>
      {children}
    </div>
  );
});
//# sourceMappingURL=GridListCell.jsx.map
