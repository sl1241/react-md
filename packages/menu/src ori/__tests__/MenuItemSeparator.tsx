import { Configuration } from "@react-md/layout";
import type { RenderResult } from "@testing-library/react";
import { fireEvent, render as baseRender } from "@testing-library/react";
import type { FC, ReactElement } from "react";

import { DropdownMenu } from "../DropdownMenu";
import { MenuItem } from "../MenuItem";
import { MenuItemSeparator } from "../MenuItemSeparator";

const Wrapper: FC = ({ children }) => (
  <Configuration disableRipple>{children}</Configuration>
);

function render(ui: ReactElement): RenderResult {
  return baseRender(ui, { wrapper: Wrapper });
}

function Test({
  horizontal,
  inset,
  vertical,
  maxHeight,
}: {
  horizontal?: boolean;
  inset?: boolean;
  vertical?: boolean;
  maxHeight?: number;
}): ReactElement {
  return (
    <DropdownMenu
      id="dropdown-menu"
      buttonChildren="Dropdown"
      horizontal={horizontal}
    >
      <MenuItem>Item 1</MenuItem>
      <MenuItemSeparator
        maxHeight={maxHeight}
        inset={inset}
        vertical={vertical}
      />
      <MenuItem>Item 2</MenuItem>
    </DropdownMenu>
  );
}

describe("MenuItemSeparator", () => {
  it("should render based on the menu's orientation", () => {
    const { getByRole, rerender } = render(<Test horizontal={false} />);
    let dropdown = getByRole("button", { name: "Dropdown" });
    fireEvent.click(dropdown);

    let separator = getByRole("separator");
    expect(separator).not.toHaveAttribute("aria-orientation");
    expect(separator).toMatchSnapshot();

    rerender(<Test key="horizontal" horizontal />);
    expect(dropdown).not.toBeInTheDocument();
    expect(separator).not.toBeInTheDocument();

    dropdown = getByRole("button", { name: "Dropdown" });
    fireEvent.click(dropdown);
    separator = getByRole("separator");
    expect(separator).toHaveAttribute("aria-orientation", "vertical");
    expect(separator).toMatchSnapshot();
  });

  it("should allow the inset styles when not rendered vertically", () => {
    const { getByRole, rerender } = render(<Test inset />);
    const dropdown = getByRole("button", { name: "Dropdown" });
    fireEvent.click(dropdown);

    const separator = getByRole("separator");
    expect(separator).toHaveClass("rmd-divider--inset");

    rerender(<Test inset horizontal />);
    expect(separator).toBeInTheDocument();
    expect(separator).not.toHaveClass("rmd-divider--inset");
  });

  it("should allow for the orientation to be overridden by the vertical prop", () => {
    const { getByRole } = render(<Test vertical />);
    const dropdown = getByRole("button", { name: "Dropdown" });
    fireEvent.click(dropdown);

    const separator = getByRole("separator");
    expect(separator).toHaveAttribute("aria-orientation", "vertical");
    expect(separator).toMatchSnapshot();
  });

  it("should allow for a custom maxHeight for vertical styles", () => {
    const { getByRole, rerender } = render(<Test maxHeight={0.5} />);
    const dropdown = getByRole("button", { name: "Dropdown" });
    fireEvent.click(dropdown);

    const separator = getByRole("separator");
    expect(separator).toMatchSnapshot();

    rerender(<Test maxHeight={0.5} vertical />);
    expect(separator).toMatchSnapshot();
  });
});
