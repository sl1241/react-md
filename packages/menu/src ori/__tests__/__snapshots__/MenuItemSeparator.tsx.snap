// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MenuItemSeparator should allow for a custom maxHeight for vertical styles 1`] = `
<li
  class="rmd-divider"
  role="separator"
/>
`;

exports[`MenuItemSeparator should allow for a custom maxHeight for vertical styles 2`] = `
<li
  aria-orientation="vertical"
  class="rmd-divider rmd-divider--vertical"
  role="separator"
  style="height: 0px;"
/>
`;

exports[`MenuItemSeparator should allow for the orientation to be overridden by the vertical prop 1`] = `
<li
  aria-orientation="vertical"
  class="rmd-divider rmd-divider--vertical"
  role="separator"
  style="height: 0px;"
/>
`;

exports[`MenuItemSeparator should render based on the menu's orientation 1`] = `
<li
  class="rmd-divider"
  role="separator"
/>
`;

exports[`MenuItemSeparator should render based on the menu's orientation 2`] = `
<li
  aria-orientation="vertical"
  class="rmd-divider rmd-divider--vertical"
  role="separator"
  style="height: 0px;"
/>
`;
